packages/core/src/Stores/client-store.js

```diff
@@ -1590,6 +1590,11 @@ export default class ClientStore extends BaseStore {
             if (!this.is_virtual) {
                 this.setPrevRealAccountLoginid(this.loginid);
             }
+            const no_cr_account = this.active_accounts.some(acc => acc.landing_company_shortcode === 'svg');
```

Comment:
we have 4 options for cr accounts. "svg", "labuan", "bvi", "vanuatu".
Just using "svg" does not suffice.

Reply:
@hirad-rewok these are MT5 shortcode, for options and multpliers we have 'mltainvest', 'svg', 'iom' , ...

Reply:
@hirad-rewok i believe we do checks just for svg for cr accounts. 🤔